demo/modules/Checkout/components/Form/index.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { Decorator as Cerebral } from 'cerebral-view-react';
+import isValidForm from 'cerebral-module-forms/helpers/isValidForm';
+
+import Input from '../Input';
+import Select from '../Select';
+import Textarea from '../Textarea';
+import Radio from '../Radio';
+import Checkbox from '../Checkbox';
+
+import classNames from 'classnames/bind'
+import styles from './styles.css';
+const cx = classNames.bind(styles);
+
+@Cerebral({
+  form: ['checkout']
+})
+class Checkout extends React.Component {
+
+  render() {
+    const {signals, form} = this.props;
+    const isValid = isValidForm(form);
+
+    return (
+      <div className='pure-g'>
+        <div className={cx('pure-u-1', 'form')}>
+          <h1>Checkout</h1>
+
+          <form className='pure-form pure-form-stacked'>
```

Comment:
FWIW,

One thing that I've been doing that I like is to not put any hard-coded 3rd party library class names in my JSX code. Instead, in styles.css I've been composing a custom class using the 3rd party libs.

E.g.

```css
// styles.css
.checkoutFormContainer {
    composes: pure-g from 'purecss/pure.css';
}

.checkoutForm {
    composes: pure-form pure-form-stacked from 'purecss/pure.css';
}

// etc.
```

```js
<div className={styles.checkoutFormContainer}>
...
```

It's been really nice not seeing any 3rd party class names in my code. ;)